tests/ftests/ftests-nocontainer.sh

```diff
@@ -6,6 +6,22 @@ AUTOMAKE_HARD_ERROR=99
 
 START_DIR=$PWD
 SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+RUNNER_LOCK_FILE="/var/run/libcgroup/github-runner.lock"
```

Comment:
I can see the lock file being needed for `ftests-nocontainer.sh` since the tests are operating on the "bare-metal" `/sys/fs/cgroup`.  But is it needed for `ftests.sh`?  `ftests.sh` generates a random container name for each run, so I don't think container foo should collide with container bar.

https://github.com/libcgroup/libcgroup/blob/main/tests/ftests/ftests.sh#L17

Reply:
Or are we doing this because the VMs are underpowered and they can't support simultaneous test runs?

Reply:
@drakenclimber Thank you so much for the review.  Yes, it's because of the underpowered VMs.